### PR TITLE
Add reference mapping to docker commands - #305 - stage 2

### PR DESCRIPTION
**Pull request by SydOps**
Add reference mapping to docker commands in comments

### CODE REVIEW

**Comment by SydOps** [comment]
The command I can't give the docker command reference is `image.insert_local`, I didn't know the same function with docker command. 

If you knew, please update me.

File: README.md [hunk 1]
```diff
@@ -139,23 +139,28 @@ image.insert_local('localPath' => [ 'Gemfile', 'Rakefile' ], 'outputPath' => '/'
...
 image.tag('repo' => 'base2', 'tag' => 'latest', force: true)
 # => ["base2:latest"]
 
 # Get more information about the Image.
+# docker command for reference: docker inspect <IMAGE.ID>
 image.json
 # => {"id"=>"67859327bf22ef8b5b9b4a6781f72b2015acd894fa03ce07e0db7af170ba468c", "comment"=>"Imported from -", "created"=>"2013-06-19T18:42:58.287944526-04:00", "container_config"=>{"Hostname"=>"", "User"=>"", "Memory"=>0, "MemorySwap"=>0, "CpuShares"=>0, "AttachStdin"=>false, "AttachStdout"=>false, "AttachStderr"=>false, "PortSpecs"=>nil, "Tty"=>false, "OpenStdin"=>false, "StdinOnce"=>false, "Env"=>nil, "Cmd"=>nil, "Dns"=>nil, "Image"=>"", "Volumes"=>nil, "VolumesFrom"=>""}, "docker_version"=>"0.4.0", "architecture"=>"x86_64"}
 
 # View the history of the Image.
+# docker command for reference, in fact it mixed the output from two commands: docker image <IMAGE.ID> and docker history <IMAGE.ID>
```
Comment (tlunter) [inline README.md]: Not sure what this means.  Looks like `Image#history` is a subset of the data returned. What are you trying to say here.

**Comment by tlunter** [comment]
@SydOps `#insert_local` doesn't actually have a docker command.  At least not how it maps currently.  It's functionality is similar to that of `docker cp host container`

File: README.md (hunk 1 above)
Comment (SydOps) [inline README.md]: The output from `image.history` has no direct docker command to be mapping. The output has the information from  two docker commands: 

```
docker image <IMAGE.ID> 
docker history <IMAGE.ID>
```

Let me think, maybe remove this comment is better.

Comment (tlunter) [inline README.md]: I agree. Seems similar to `#insert_local` that doesn't really have a 1-to-1 mapping.

**Comment by SydOps** [comment]
updated, please confirm

**Pull request merged**